warehouse/banners/models.py

```diff
@@ -31,6 +31,7 @@ class Banner(db.Model):
     link_url: Mapped[str]
     link_label: Mapped[str] = mapped_column(default=DEFAULT_BTN_LABEL)
     fa_icon: Mapped[str] = mapped_column(default=DEFAULT_FA_ICON)
+    dismissable: Mapped[bool] = mapped_column(default=True)
```

Comment:
question: we have `bool_true` as a type in `warehouse.utils.db.types` that uses `server_default` which would set the value server-side if we used an `insert` directly, vs `default` here that sets the value in Python-land. Was this choice intentional?
If changed, need to re-generate the migration so that the value is passed to the database.

Reply:
Interestingly I actually meant to set the default to `false` here as I was going back and forth on what the default should be, so nice catch! I'm OK with setting this via `server_default`!